CHANGELOG.md

```diff
@@ -4,6 +4,105 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 8.0.0-rc.0
+
+This is the first release candidate of Sentry JavaScript SDK v8.
+
+We also recommend, reading the detailed
+[migration guide](https://docs.sentry.io/platforms/javascript/migration/v7-to-v8/) in the docs.
+
+### Version Support
+
+The Sentry JavaScript SDK v8 now supports Node.js 14.8.0 or higher. This applies to @sentry/node and all of our
+node-based server-side sdks (@sentry/nextjs, @sentry/remix, etc.).
+
+The browser SDKs now require ES2018+ compatible browsers. New minimum browser versions:
+
+Chrome 63 Edge 79 Safari/iOS Safari 12 Firefox 58 Opera 50 Samsung Internet 8.2
+
+For more details, please see the
+[version support section in migration guide](https://github.com/getsentry/sentry-javascript/blob/8.0.0-beta.1/MIGRATION.md#1-version-support-changes).
+
+### Important Changes
+
+- **feat(svelte): Add Svelte 5 support (#11807)**:
+
+We now officially support Svelte 5.
+
+- **feat(node): Support Node 22 (#11871)**:
+
+We now officially support Node 22.
+
+- **fix(node): Fix nest.js error handler (#11874)**
```

Comment:
I would move all these fixes down to "Other Changes" - important here is really the big, breaking stuff only!

Reply:
I added all important changes from the last beta releases plus Node 22 support and the one with Nest.js as the Nest.js fix is a breaking change, right?